app/templates/job-layout.html

```diff
@@ -12,18 +12,17 @@
         <div class="col-lg-12">
             <p>
                 <strong>Wikimedia Deutschland – Gesellschaft zur Förderung Freien Wissens e. V.</strong> is a non-profit organisation based in Berlin, Germany.
-                With more than 80 employees and about 50.000 members, it is the oldest and largest of more than 40 independent chapters of
-                the international Wikimedia movement. The most well-known Wikimedia project, the free online encyclopedia Wikipedia, is used
+                With more than 100 employees and about 65.000 members, it is the oldest and largest of 37 independent chapters of
```

Comment:
65.000 makes me shiver, to me this means 65.
Perhaps it should just be written as 65000?

Reply:
Standard of English writing says "65,000". The German way of writing it is "65.000".

Reply:
Indeed, so as this is written in English perhaps it should be with a ","?
I still think it would be fine without either to be honest.

Reply:
changed to a comma.
sorry, I've blindly copied stuff without checking such details